src/manager.ts

```diff
@@ -218,4 +219,35 @@ export class Manager {
         this.extension.completer.command.getCommandsTeX(filePath)
         this.extension.completer.reference.getReferencesTeX(filePath)
     }
+
+    findBinary(config: string, candidates: string[]) {
+        const configuration = vscode.workspace.getConfiguration('latex-workshop')
+        const configCommand = configuration.get(config) as string
+        if (fs.existsSync(configCommand)) {
+            return
+        }
+        hasbin(configCommand, result => {
+            if (result) {
+                return
+            }
+            let found = false
+            for (const candidate of candidates) {
+                if (fs.existsSync(candidate)) {
+                    found = true
+                    vscode.window.showWarningMessage(`'latex-workshop.${config}' value '${configCommand}' is invalid. \
```

Comment:
Might be nicer here to make one of the dropdown menus (as we do for citation browser) so the user can scroll the paths and pick one.

So flow could be:

1. Warning message - the current config value is invalid, but other binaries were found. Do you want to select one of them? 

2. If yes, show the citation browser style window with text 'Select a path for the BINARY_NAME binary to replace your workspace* settings.